.schema/config.schema.json

```diff
@@ -1095,6 +1095,24 @@
             "key_length": {
```

Comment:
Please update the other defaults as well

Reply:
Should we make the memory parameter a string as well (like the dedicated memory)? Nobody will be able to understand it currently without a calculator.

Reply:
YES!!